earth_enterprise/rpms/opengee-fusion/post-install.sh

```diff
@@ -23,6 +23,9 @@ set +x
 # Versions and user names:
 GEE="Google Earth Enterprise"
 GEEF="$GEE Fusion"
+CHKCONFIG="/sbin/chkconfig"
+INITSCRIPTUPDATE="/usr/sbin/update-rc.d"
```

Comment:
Minor: Can we use some word separators in variable names? I have to stare too long to read these.

Reply:
Because these get used in both server and fusion, they should migrate to gevars.sh, and yes, we_can_add a separator when doing so, it does not hurt at all...